packages/common/src/extensionDependencies.ts

```diff
@@ -4,4 +4,5 @@ export const extensionDependencies = [
   "scalameta.metals",
   "ms-python.python",
   "mrob95.vscode-talonscript",
+  "jrieken.vscode-tree-sitter-query",
```

Comment:
I'm leaning or more towards just including our own missing file extensions instead of having to add additional extension dependencies.

Reply:
Agreed. Tho wrt .scm specifically, @auscompgeek makes a good point that we may want to avoid that one as it is more commonly used for Scheme. But for .talon and the others I think it makes sense for us to just register them